docs/tutorials/tutorial-rollup.md

```diff
@@ -24,18 +24,21 @@ sidebar_label: Aggregate data with rollup
   -->
 
 
-Apache Druid can summarize raw data at ingestion time using a process we refer to as "rollup". Rollup is a first-level aggregation operation over a selected set of columns that reduces the size of stored data.
+Apache Druid&circledR; can summarize raw data at ingestion time using a process known as "rollup." [Rollup](../ingestion/rollup.md) is a first-level aggregation operation over a selected set of columns that reduces the size of stored data.
 
-This tutorial will demonstrate the effects of rollup on an example dataset.
+The tutorial demonstrates how to apply rollup at ingestion and shows the effect of rollup at query time. See [ingesting with rollup](https://druid.apache.org/docs/latest/multi-stage-query/concepts/#rollup) to learn more. 
```

Comment:
```suggestion
This tutorial demonstrates how to apply rollup during ingestion and highlights its effects during query execution. 
```
Remove this line as it doesn't apply to the preceding sentence. Replace the link on line 27 with this link (https://druid.apache.org/docs/latest/multi-stage-query/concepts/#rollup) but use the relative path.

Reply:
removed "See ingesting with rollup to learn more." and added "The examples in the tutorial use the multi-stage query (MSQ) task engine to executes SQL statements." in its place